src/ts/player-config.d.ts

```diff
@@ -390,6 +390,11 @@ declare namespace bitmovin {
        * HLS content can easily and for free be generated using our encoding solution bitcodin.
        */
       hls?: string;
+      /**
+       * An URL to a Microsoft Smooth Streaming Manifest (normally ends with .ism/Manifest but can also be a .xml)
+       * Supported by player version 7.5
```

Comment:
Please use the `@since` JSDoc/TypeDoc tag instead of a plain text sentence: `@since v7.5`